src/octoprint/plugins/eventmanager/templates/eventmanager_settings.jinja2

```diff
@@ -4,15 +4,15 @@
 </div>
 <div class="row-fluid">
     <div class="span4">{{ _('Event') }}</div>
-    <div class="span6">{{ _('Command') }}</div>
+    <div class="span6">{{ _('Name') }}</div>
     <div class="span2 btn-group">
         <button class="btn btn-mini" data-bind="click: addEvent"><i class="fa fa-plus"></i> {{ _('Add') }}</button>
     </div>
 </div>
 <div data-bind="foreach: settingsViewModel.settings.plugins.eventmanager.subscriptions" class="row-fluid">
     <div class="row-fluid">
         <div class="span4" data-bind="text: event"></div>
-        <div class="span6" data-bind="text: command"></div>
+        <div class="span6" data-bind="attr: {title: command}, text: name"></div>
```

Comment:
This is missing the mentioned client-side fallback.

```suggestion
        <div class="span6" data-bind="attr: {title: command}, text: name() ? name() : command()"></div>
```

Reply:
this fallback doesn't seem to work with upgraded migration. see discussion on Discord starting [here](https://discord.com/channels/704958479194128507/705047010641838211/1096931547413758033) @foosel.